src/ducks/devtools/Queries.jsx

```diff
@@ -48,15 +71,33 @@ const QueryData = ({ data, doctype }) => {
   const handleShowTable = useCallback(() => setShowTable(value => !value), [
     setShowTable
   ])
+
+  const [results, setResults] = useState(null)
+  const [search, setSearch] = useState('')
+  const handleSearch = useCallback(
+    ev => {
+      const searchValue = ev.target.value
+      const matcher = searchValue !== '' ? makeMatcher(searchValue) : null
+      const results = matcher ? storeData.filter(matcher) : null
+      setSearch(searchValue)
+      setResults(results)
+    },
+    [storeData]
+  )
+
+  const viewData = results || storeData
+
   return (
     <>
       Table:{' '}
       <input type="checkbox" onClick={handleShowTable} checked={showTable} />
       <br />
+      Search: <input type="text" onChange={handleSearch} value={search} />
```

Comment:
adding a placeholder telling us how to search can be useful since we split on ";" and ":"

Reply:
:+1: Done, thanks